Week1/2.txt

```diff
@@ -0,0 +1,15 @@
+2)Write a python script to enter any number and print the sum of its digit.
+def getSum(n):
+    
+    sum = 0
+    for digit in str(n): 
+      sum += int(digit)      

```

Comment:
why you use + symbol ?

Reply:
The + symbol is used as the addition operator to perform arithmetic addition.